pyramid_oereb/contrib/sources/document.py

```diff
@@ -76,21 +76,23 @@ def __init__(self, **kwargs):
         if self._parser.host_url is None:
             raise AssertionError('host_url has to be defined')
 
-    def read(self, params, geolink_id):
+    def read(self, params, geolink_id, oereblex_params):
```

Comment:
```suggestion
    def read(self, params, geolink_id, oereblex_params=None):
```
To avoid passing `None` explicitely in tests for instance